File: src/app/model/loaisanpham.ts
export class lsanpham {
    _id: number;
    tendanhmuc: string;
    thuonghieu: Array<string>;
    dactrung: Array<object>;
    tieuchidanhgia: Array<string>;
   
    constructor( _id: number,  tendanhmuc: string, thuonghieu: Array<string>, dactrung: Array<object>, tieuchidanhgia: Array<string>) {
    	this._id = _id;
        this.tendanhmuc=tendanhmuc;
    	this.thuonghieu=thuonghieu;
        this.dactrung=dactrung;
        this.tieuchidanhgia=tieuchidanhgia;
    }
}

File: dienmayxanhapi/dienmayxanhapi/Model/loaisanpham.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Driver;

namespace dienmayxanhapi.Model
{
    public class loaisanpham
    {
        public int? _id { get; set; }
        public string tendanhmuc { get; set; }
        public List<string> thuonghieu { get; set; }
        public IList<dynamic> dactrung { get; set; }
        public List<string> tieuchidanhgia { get; set; }

    }
}
